src/batch/src/executor/join/hash_join.rs

```diff
@@ -2455,16 +2464,38 @@ mod tests {
         }
     }
 
-    fn is_data_chunk_eq(left: &DataChunk, right: &DataChunk) -> bool {
+    fn is_data_chunk_bag_eq(left: &DataChunk, right: &DataChunk) -> bool {
```

Comment:
What is the meaning of `bag`? Perhaps add some docs

Reply:
Same as `query III rowsort` we use in the sql logic test, i.e. we don't care about the order of the result.

Reply:
Let me give it a better name `compare_data_chunk_with_rowsort`